locales/nb_NO/pcsx2_Main.po

```diff
@@ -1198,33 +1134,31 @@ msgstr "Velg en mappe for %sinnstillinger"
 msgid "Settings"
 msgstr "Innstillinger"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:86
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:84
 msgid "Language selector"
 msgstr "Språkvelger"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:89
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:87
 msgid ""
 "Change the language only if you need to.\n"
 "The system default should be fine for most operating systems."
 msgstr ""
 "Skift språket bare hvis du trenger det.\n"
-"Systemet sin standardinnstilling bør gå greit for de fleste operativsystemer."
+"Systemet sin standardinnstilling burde være greit for de fleste operativsystemer."
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:95
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:93
 msgid "Welcome to PCSX2!"
 msgstr "Velkommen til PCSX2!"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
-#, fuzzy
-msgid "Configuration Guide"
-msgstr "Oppsettsanvisninger (Online)"
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:104
+msgid "Configuration Guides (online)"
```

Comment:
This string was changed recently to `Configuration Guide`. I'm guessing you were working from an older version of the file, which is why it reverted back.